templates/terraform/examples/app_engine_standard_app_version.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_project" "my_project" {
+  name       = "tf-test-project"
+  project_id = "<%= ctx[:vars]['project_id'] %>"
+  org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_app_engine_application" "app" {
+  project     = "${google_project.my_project.project_id}"
+  location_id = "us-central"
+}
+
+
+resource "google_app_engine_app_version" "<%= ctx[:primary_resource_id] %>" {
+  version_id = "v31"
+  project = "${google_app_engine_application.app.project}"
+  service = "default"
+  runtime = "nodejs10"
+  api_config = {}
+  endpoints_api_service = {}
+  entrypoint {
+    shell = "node index.js"
+  }
```

Comment:
```suggestion
  }
}
```

Reply:
Thank you very much Riley for your support. I have made the changes. Can you also tell me if I can run those tests locally to validate. I did do compile and test the newly built provider locally, but I don't think I am running the go tests for the provider.

Reply:
Instructions for running the provider tests are at https://github.com/terraform-providers/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests, they'll be ran directly from the provider directory.

Make sure to include a pattern, probably `TestAccAppEngineStandardAppVersion_`. All of our tests are acceptance tests, and running other product suites wouldn't be useful.

Reply:
Thanks for the response. I will do.
Also one more question. When we release this new resource for App Version, would it be compatible with Terraform 0.11? Or would I need 0.12+?

Reply:
It'll be compatible with both!